docs/cloud/_category_ .yml

```diff
@@ -0,0 +1,4 @@
+position: 3
+label: "Cloud"
```

Comment:
Hey @Niennienzz - I can't get this to be with capital "C" on the left menu 

Also how can I add a "cloud" icon to the menu item ? 

Thanks!

Reply:
This should be fixed now. I am using the `Getting Started` icon temporarily, will get a new icon from the designer.

Reply:
@Niennienzz - FYI - added a cloud icon meanwhile